friends/friendModel.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+const friendSchema = new mongoose.Schema({
+	firstName: { type: String, required: true },
+	lastName: { type: String, required: true },
+	age: { type: Number, required: true, min: 1, max: 120 },
+	createdOn: { type: Date, required: true, default: Date.now },
+});
+
+const friendModel = mongoose.model('friend', friendSchema);
```

Comment:
Instead of just having 'friend' I would put 'Friend'. It's convention, because mongo will automatically make the collection name the lowercase, plural version of the name you give. It's not wrong though